File: app/src/main/java/andi/android/madegdk/ui/launcher/MainActivity.java
package andi.android.madegdk.ui.launcher;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import andi.android.madegdk.ui.home.HomeActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, HomeActivity.class));
        finish();
    }

}

File: app/src/main/java/andi/android/madegdk/ui/launcher/MainActivity.kt
package andi.android.madegdk.ui.launcher

import andi.android.madegdk.database.FavoriteMovieHelper
import andi.android.madegdk.database.FavoriteTvSeriesHelper
import andi.android.madegdk.ui.home.HomeActivity
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    private var favoriteTvSeriesHelper: FavoriteTvSeriesHelper? = null
    private var favoriteMovieHelper: FavoriteMovieHelper? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        favoriteTvSeriesHelper = FavoriteTvSeriesHelper.getInstance(applicationContext)
        favoriteMovieHelper = FavoriteMovieHelper.getInstance(applicationContext)

        favoriteTvSeriesHelper?.open()
        favoriteMovieHelper?.open()

        startActivity(Intent(this, HomeActivity::class.java))
        finish()
    }

    override fun onDestroy() {
        super.onDestroy()
//        favoriteTvSeriesHelper?.close()
    }

}